### PR TITLE
Replace Batcher* with Batcher& in UpdatePrimitives

### PR DESCRIPTION
**Pull request by reichlfl**
Other methods (such as Draw) already receive Batcher&, so this way
we're consistent across our drawing methods.

### CODE REVIEW

File: src/OrbitGl/TimerTrack.cpp [hunk 1]
```diff
@@ -252,7 +252,7 @@ void TimerTrack::DoUpdatePrimitives(Batcher* batcher, TextRenderer& text_rendere
   draw_data.min_tick = min_tick;
   draw_data.max_tick = max_tick;
 
-  draw_data.batcher = batcher;
+  draw_data.batcher = &batcher;
```
Comment (karupayun) [inline src/OrbitGl/TimerTrack.cpp:255]: shouldn't this be a reference also?

**Review by karupayun — APPROVED** [review]
LGTM! Thanks for this!

File: src/OrbitGl/TimerTrack.cpp (hunk 1 above)
Comment (reichlfl) [reply src/OrbitGl/TimerTrack.cpp:255]: Yeah, but then `DrawData` needs a proper constructor (you can't leave a reference uninitialized), and I didn't want to start going there and refactoring `GetDrawData`...

Comment (karupayun) [reply src/OrbitGl/TimerTrack.cpp:255]: got it!

**Review by vickyliu-go4it — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**